testsuite/mergeinstances-duplicate-entrylayers/a.osl

```diff
@@ -0,0 +1,17 @@
+shader a
+(
+    float frequency = 1,
+
```

Comment:
Totally not critical, but is there a reason why every second line is empty?

Reply:
Fixed